src/stars.css

```diff
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
```

Comment:
Make sure that the last star in each row doesn't have margin-right
![image](https://github.com/mate-academy/layout_stars/assets/96409563/f6351688-b1c9-49ae-b7b1-bc1e0ec632d2)